app/command.go

```diff
@@ -252,7 +252,10 @@ func (a *App) mentionsToTeamMembers(message, teamId string) model.UserMentionMap
 		}(mention)
 	}
 
-	wg.Wait()
+	go func() {
```

Comment:
Oops, I forgot to think through the required `close` in my channel simplifications. 0/5, but this doesn't need to be in a goroutine: you can block on the `sync.WaitGroup` and then immediately close the channel, finally iterating over it.

Reply:
I forgot, too :see_no_evil:  I agree there is no need to put it in a goroutine, but I think this does not hurt readability, while still leveraging concurrency, as the loop can start consuming the mentions.

Reply:
Thinking through this a second time, I think I might be more than 0/5 on this. My rationale would be:
* I don't think concurrency improves performance here. The actual act of consuming from a pre-buffered channel and writing to the map is going to be negligible. The goroutine itself won't be free, and may outweigh any potential advantage given the average number of mentions for a given message.
* A few extra lines isn't a big deal, but as a reader of the code, I now need to mentally fork on this goroutine and reason through the potential edge cases. The change in complexity from the previous linearity at this point in the code is non-trivial.

Reply:
I agree that the improvement in performance will be negligible (we are probably talking about a couple of mentions in the average use-case), and the goroutine may even outweight that.

And after a bit more of thought, it's true that the complexity added to the reader may be greater than I have anticipated, so I will remove the goroutine. Thank you for the thorough rationale!